pkgs/by-name/ei/eigenlayer-cli/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGo121Module,
+  fetchFromGitHub,
+}:
+buildGo121Module rec {
```

1.21 is the current default of buildGoModule and is the preferred way (:

```suggestion
  buildGoModule,
  fetchFromGitHub,
}:
buildGoModule rec {
```

---

Fixed!